api/src/wfl/service/google/pubsub.clj

```diff
@@ -0,0 +1,186 @@
+(ns wfl.service.google.pubsub
+  "Wrappers for Google Cloud Pub/Sub REST APIs.
+   See https://cloud.google.com/pubsub/docs/reference/rest"
+  (:require [clj-http.client :as http]
+            [clojure.data.json :as json]
+            [clojure.string :as str]
+            [wfl.once :as once]
+            [wfl.util :as util]))
+
+(defn ^:private json-body [response]
+  (-> response :body (or "null") util/parse-json))
+
+(def ^:private pubsub-url
+  (partial str "https://pubsub.googleapis.com/v1/"))
+
+;; Google Cloud Pub/Sub Topics
+;; https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics
+
+(defn create-topic
+  "Create a Pub/Sub topic in `project` with the specified `topic-id`. Returns a
+   newly created topic name in the form \"/projects/PROJECT/topics/TOPIC-ID\".
+
+   Parameters
+   ----------
+   project  - Google Cloud Project to create the Pub/Sub topic in.
+   topic-id - unique identifier for the topic
+
+   Example
+   -------
+     (let [topic (create-topic \"broad-gotc-dev\" \"my-topic\")]
+       #_(;; use topic ))"
```

Comment:
like this docstring style, reminds me of the [numpy docs](https://numpydoc.readthedocs.io/en/latest/format.html).

Reply:
To each his own.